src/twisted/web/newsfragments/9383.bugfix

```diff
@@ -0,0 +1 @@
+in templating module, slot data stack is badly cleaned after flattening a node tree
```

Comment:
Please start this with the full name of the public class/module/function which is affected...and end the sentence with a full stop :)

Reply:
done